.rubocop.yml

```diff
@@ -15,17 +17,14 @@ AllCops:
     - '.git/**/*'
 
 IndentationConsistency:
-  EnforcedStyle: 'rails'
+  EnforcedStyle: 'indented_internal_methods'
```

Comment:
Another change in [RuboCop version 0.72](https://github.com/rubocop-hq/rubocop/releases/tag/v0.72.0).